module/item/item.mjs

```diff
@@ -1240,11 +1240,7 @@ export class HeroSystem6eItem extends Item {
                             const token = tokens[0];
                             const velocity = calculateVelocityInSystemUnits(item.actor, token);
 
-                            item.system.ocvEstimated = (
-                                ocv +
-                                parseInt(cslSummary.ocv) +
-                                parseInt(velocity / 10)
-                            ).signedString();
+                            item.system.ocvEstimated = String(ocv + parseInt(cslSummary.ocv) + parseInt(velocity / 10));
```

Comment:
It's usual to use String as a function to coerce to a string,  I'm not sure if this is done elsewhere in this code base. For the sake of consistency, is there any advantage over using a template literal `` `${ocv + parseInt(cslSummary.ocv) + parseInt(velocity / 10)}` ``?

(same comment for the other use of this pattern)

Reply:
nah no advantage either way in practice